demo/demo_args.c

```diff
@@ -1125,6 +1128,19 @@ int demo_parse_argv(cmdline_args_t *parsed_args, int argc, char *argv[]) {
         case 305:
             parsed_args->alternative_logger = true;
             break;
+        case 306: {
+            if(!optarg) {
+                demo_log(ERROR, "Missing identity string");
+                goto finish;
+            }
```

Comment:
This would be redundant if you used `required_argument`

Reply:
Oh, I thought required argument does something else :D (as in you need to provide it to the executable in order for it to be happy, it is more like "requires value", right?)